crates/thetawave_interface/src/spawnable.rs

```diff
@@ -72,6 +72,13 @@ pub enum MobType {
 }
 
 impl MobType {
+    pub fn is_boss(&self) -> bool {
+        match self {
+            MobType::Enemy(EnemyMobType::Repeater) => true,
```

Comment:
How is this different from `Query<BossComponent>`? *gulp*

Reply:
@LordDeatHunter pointed this out too. I'm going to change this.